### ISSUE TITLE
appimage-run: $OWD variable not set

### ISSUE THREAD

**Issue by gvegidy** [NixOS/nixpkgs#203330]
### Describe the bug
When running an AppImage binary with `appimage-run`, the $OWD environment variable is not populated.

OWD is one of the environment variables that are usually populated by the AppImage runtime. Several AppImages expect it to be set to work correctly. OWD contains the path to working directory at the time the AppImage is called
https://docs.appimage.org/packaging-guide/environment-variables.html

### Steps To Reproduce
1. NixOS 22.05
2. Install appimage-run
3. `appimage-run owdtest.AppImage`
4. Result: `line 5: OWD: unbound variable`

owdtest is attached, it has to be `gunzip`'ed:
[owdtest.AppImage.gz](https://github.com/NixOS/nixpkgs/files/10099048/owdtest.AppImage.gz)

### Expected behavior
`ls` is executed for the current working directory, no unbound variable error.

### Notify maintainers
@symphorien
@prusnak

### Metadata
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.79, NixOS, 22.05 (Quokka), 22.05.4338.695b3515251`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.1`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by symphorien** [comment]
so it's just a question of adding `export OWD=$(readlink -f .)` inside appimage-run? If so, would you mind opening a PR?

**Comment by gvegidy** [comment]
Maybe? I must admit I'm completely new to Nix. 

I just want to publish an AppImage that runs on as many distros as possible. NixOS was one of the distros I tested it with. So I installed NixOS today and during testing I saw this issue. So I thought it would be helpful to report this issue.

I don't have a development environment set up for NixOS and would have to read up how to build it, while I would prefer to test my AppImage on some more platforms instead.
